From akpm@linux-foundation.org Thu Apr 26 00:58:02 2007
From: Andrew Morton <akpm@linux-foundation.org>
Date: Thu, 26 Apr 2007 00:12:10 -0700
Subject: sysfs: bin.c printk fix
To: greg@kroah.com
Cc: akpm@linux-foundation.org
Message-ID: <200704260712.l3Q7CAXq023462@shell0.pdx.osdl.net>


From: Andrew Morton <akpm@linux-foundation.org>

fs/sysfs/bin.c: In function 'read':
fs/sysfs/bin.c:77: warning: format '%zd' expects type 'signed size_t', but argument 4 has type 'int'



Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 fs/sysfs/bin.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/fs/sysfs/bin.c
+++ b/fs/sysfs/bin.c
@@ -59,7 +59,7 @@ read(struct file * file, char __user * u
 	if (copy_to_user(userbuf, buffer, count))
 		return -EFAULT;
 
-	pr_debug("offs = %lld, *off = %lld, count = %zd\n", offs, *off, count);
+	pr_debug("offs = %lld, *off = %lld, count = %d\n", offs, *off, count);
 
 	*off = offs + count;
 
